src/Aquarius.ONE.ClientSDK/Utilities/PlatformEnvironmentHelper.cs

```diff
@@ -44,6 +44,16 @@ public static PlatformEnvironment GetPlatformEnvironment(string name)
                     case "PRODUCTION":
                     case "4":
                         return PlatformEnvironmentHelper.GetPlatformEnvironment(EnumPlatformEnvironment.AqiUSProduction);
+                    case "AQIEUPRODUCTION":
+                    case "AQI EU PRODUCTION":
+                    case "ONE EU PRODUCTION":
+                    case "ONEEUPRODUCTION":
+                    case "EUPRODUCTION":
+                    case "EU PRODUCTION":
+                    case "EU-PRODUCTION":

```

Comment:
this looks like it is extra?  We don't have the dash notation for any other environments.

Reply:
it was just added in case someone wanted to instantiate a client that way @IsaakMalers-AQI